include/solvers/postprocessing_velocities.h

```diff
@@ -38,36 +38,49 @@
 using namespace dealii;
 
 /**
- * @brief Class that calculates the
- * time-averaged velocities and pressure \f$(\langle u \rangle, \langle v
- * \rangle, \langle w \rangle, \langle p \rangle)\f$ and the independent
- * components of the Reynolds stresses tensor \f$(\langle u'u' \rangle, \langle
- * v'v' \rangle, \langle w'w' \rangle, \langle u'v' \rangle, \langle v'w'
- * \rangle, \langle w'u' \rangle)\f$. The generated vectors are displayable of
- * visualization software.
+ * @brief Calculates the time-averaged velocities and pressure \f$(\langle u
+ * \rangle, \langle v \rangle, \langle w \rangle, \langle p \rangle)\f$ and the
+ * independent components of the Reynolds stresses tensor \f$(\langle u'u'
+ * \rangle, \langle v'v' \rangle, \langle w'w' \rangle, \langle u'v' \rangle,
+ * \langle v'w' \rangle, \langle w'u' \rangle)\f$. The generated vectors can be
+ * displayed using common visualization software.
```

Comment:
```suggestion
 * \langle v'w' \rangle, \langle w'u' \rangle)\f$. 
```
I think we can remove that statement. We always output to vtu files anyway.